chemprop/featurizers/atom.py

```diff
@@ -133,7 +136,117 @@ def num_only(self, a: Atom) -> np.ndarray:
+            ],
+        )
+
+    @classmethod
+    def v1(cls, max_atomic_num: int = 100):
+        """The original implementation used in Chemprop V1 [1]_, [2]_.
+
+        Parameters
+        ----------
+        max_atomic_num : int, default=100
+            Include a bit for all atomic numbers in the interval `[1, max_atomic_num]`
+
+        References
+        -----------
+        .. [1] Yang, K.; Swanson, K.; Jin, W.; Coley, C.; Eiden, P.; Gao, H.; Guzman-Perez, A.; Hopper, T.;
+        Kelley, B.; Mathea, M.; Palmer, A. "Analyzing Learned Molecular Representations for Property Prediction."
+        J. Chem. Inf. Model. 2019, 59 (8), 3370–3388. https://doi.org/10.1021/acs.jcim.9b00237
+        .. [2] Heid, E.; Greenman, K.P.; Chung, Y.; Li, S.C.; Graff, D.E.; Vermeire, F.H.; Wu, H.; Green, W.H.; McGill,
+        C.J. "Chemprop: A machine learning package for chemical property prediction." J. Chem. Inf. Model. 2024,
+        64 (1), 9–17. https://doi.org/10.1021/acs.jcim.3c01250
+        """
+
+        return cls(
+            atomic_nums=list(range(1, max_atomic_num + 1)),
+            degrees=list(range(6)),
+            formal_charges=[-1, -2, 1, 2, 0],
+            chiral_tags=list(range(4)),
+            num_Hs=list(range(5)),
+            hybridizations=[
+                HybridizationType.SP,
+                HybridizationType.SP2,
+                HybridizationType.SP3,
+                HybridizationType.SP3D,
+                HybridizationType.SP3D2,
+            ],
+        )
+
+    @classmethod
+    def organic(cls):
+        r"""A specific parameterization intended for use with organic or drug-like molecules.
+
+        This parameterization features:
+            1. includes an atomic number bit only for H, B, C, N, O, F, Si, P, S, Cl, Br, and I atoms
+            2. a hybridization bit for :math:`s, sp, sp^2` and :math:`sp^3` hybridizations.
+        """
+
+        return cls(
+            atomic_nums=[1, 5, 6, 7, 8, 9, 14, 15, 16, 17, 35, 53],
+            degrees=list(range(6)),
+            formal_charges=[-1, -2, 1, 2, 0],
+            chiral_tags=list(range(4)),
+            num_Hs=list(range(5)),
+            hybridizations=[
+                HybridizationType.S,
+                HybridizationType.SP,
+                HybridizationType.SP2,
+                HybridizationType.SP3,
+            ],
+        )
+
+
+class AtomFeatureMode(EnumMapping):
+    """The mode by which a atom should be featurized into a `MolGraph`"""
+
+    DEFAULT = auto()
+    V1 = auto()
+    ORGANIC = auto()
+
+
+def get_MultiHotAtomFeaturizer(mode: str | AtomFeatureMode) -> MultiHotAtomFeaturizer:
+    """Gets an multi-hot atom featurizer given the name of the `AtomFeatureMode`.
+    Parameters
+    ----------
+    mode : str | AtomFeatureMode
+        The name of the multi-hot atom featurization scheme.
+    Returns
+    -------
+    MultiHotAtomFeaturizer
+        The multi-hot atom featurizer module.
+    """
```

Comment:
```suggestion
def get_multi_hot_atom_featurizer(mode: str | AtomFeatureMode) -> MultiHotAtomFeaturizer:
    """Build the corresponding multi-hot atom featurizer."""
```

- IMO this function and its signature are self-explanatory
- function names use snake-case and class names use camel-case